### PR TITLE
DT-2101 publish from circle

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -41,7 +41,9 @@ jobs:
             - gradle-{{ checksum "build.gradle.kts" }}
             - gradle-
       - run:
-          command: ./gradlew -Dorg.gradle.jvmargs="--illegal-access=permit" -Dkotlin.daemon.jvm.options="--illegal-access=permit" publish
+          command: |
+            ORG_GRADLE_PROJECT_signingKey=$(echo -e "$ORG_GRADLE_PROJECT_signingKey")
```
Comment (mikehalmamoj) [inline .circleci/config.yml:45]: Replaces `\n` in the private key env var with new lines.

File: README.md [hunk 2]
```diff
@@ -6,23 +6,16 @@ A helper library providing utilities for using `amazon-sqs-java-messaging-lib`
 
 ## Publishing to Maven Central
 
-[This guide](https://central.sonatype.org/publish/publish-guide/) was used to publish to Maven Central. However please
-note that the Gradle plugin used in that document - `maven` - is out of date and we had to switch to
-the [maven-publish plugin](https://docs.gradle.org/current/userguide/publishing_maven.html) instead.
+[This guide](https://central.sonatype.org/publish/publish-guide/) was used to publish to Maven Central. However please note that the Gradle plugin used in that document - `maven` - is out of date and we had to switch to the [maven-publish plugin](https://docs.gradle.org/current/userguide/publishing_maven.html) instead.
```
Comment (mikehalmamoj) [inline README.md:9]: Most of these updates are just white space.

File: README.md [hunk 3]
```diff
@@ -31,48 +24,34 @@ Note that this is the place to look for clues if the publish fails. ^^^
...
 
 * Generate a new key - follow the [Sonatype guide](https://central.sonatype.org/publish/requirements/gpg/).
-* Export the private key - google for `gpg export private key` and you should find several guides for
-  using `gpg --export-secret-keys`.
-* Delete the environment variables `ORG_GRADLE_PROJECT_signingKey` and `ORG_GRADLE_PROJECT_signingPassword` from
-  the [Circle CI env vars page](https://app.circleci.com/settings/project/github/ministryofjustice/hmpps-spring-boot-sqs/environment-variables)
-* Recreate the environment variables where `ORG_GRADLE_PROJECT_signingKey` contains the private key
-  and `ORG_GRADLE_PROJECT_signingPassword` contains the passphrase  
+* Export the private key to a file - google for `gpg export private key` and you should find several guides for using `gpg --export-secret-keys`.
+* To allow the private key to be inserted into Circle, convert the newlines in the private key to `\n` with command (assuming the private key is stored in file `private.key`): `cat private.key | sed -E ':a;N;$!ba;s/\r{0,1}\n/\\n/g'`
```
Comment (mikehalmamoj) [inline README.md:55]: This is new.

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**